### PR TITLE
Fixed linting issues in the assigned directories-Ritam

### PR DESCRIPTION
**Pull request by ritam2358**
Used ESLint to fix the linting problems in Footers, Navbars and UI folders.

### CODE REVIEW

**Comment by ritam2358** [comment]
fixes #5

File: .eslintrc.js [hunk 1]
```diff
@@ -1,24 +0,0 @@
-module.exports = {
```
Comment (shubhamkorde) [inline .eslintrc.js:1]: Why was this file deleted?

**Comment by ritam2358** [comment]
It was the eslint config file that I had used for the linting purposes. It was created later and not part of the codebase, hence I deleted it. Furthermore, it was according to eslint version 7 not 5 because of the issues I had on my side.

File: .eslintrc.js (hunk 1 above)
Comment (shubhamkorde) [reply .eslintrc.js:1]: The recently merged PR must be showing conflicts now. Update your PR by resolving the conflicts.
